python/test/test_hail/table/test_table.py

```diff
@@ -554,3 +554,35 @@ def test_write_stage_locally(self):
         t.write(f, stage_locally=True)
         t2 = hl.read_table(f)
         self.assertTrue(t._same(t2))
+
+    def test_read_back_same_as_exported(self):
```

Comment:
this test could be smaller, probably. This is a test we 100% should have, but it might be a bit more future-proof to implement as a Table.parallelize with the full schema we expect to be able to parse, and hand-constructed examples of things we want to test.